src/configParamsJSON/lookupConfigParams.ts

```diff
@@ -105,29 +105,13 @@ export default {
               ],
             },
             {
+              // move this setting then update to TCL?
```

Comment:
Is this a `todo`?

Reply:
no just accidentally left it. I'll remove